charts/certmanager/templates/cert-manager-installation.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-cert-manager-operator
+  namespace: cert-manager-operator
+spec:
+  channel: "{{ .Values.operatorChannel }}"
+  installPlanApproval: Automatic
+  name: openshift-cert-manager-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cert-manager-operator
+  namespace: cert-manager-operator
+spec:
+  targetNamespaces:
+  - cert-manager-operator
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  name: cluster
+  annotations:
+{{- if eq .Values.installerType "argocd" }}
```

Comment:
There is no harm adding both annotations. Though, we probably only need the Argo CD annotation if we are going to delegate the operator install to the VP framework